examples/adjustwindow_updateviewer_saveimage.py

```diff
@@ -0,0 +1,73 @@
+from napari.settings import get_settings
+import time
+import napari
+from napari._qt.qthreading import thread_worker
+from skimage import data
+from skimage.morphology import ball, octahedron
+import matplotlib.pyplot as plt
+
+
+def make_screenshot(viewer):
+    img = viewer.screenshot(canvas_only=True, flash=False)
+    plt.imshow(img)
+    plt.axis("off")
+    plt.show()
+
+
+get_settings().application.window_position = (900, 300) # hopefully better option to do this
+viewer = napari.Viewer()
+viewer.window.resize(600, 600) 
+#viewer.window.position(600, 600) # is it possible that this will replace get_settings().application.window_position = (900, 300)?
+
+viewer.window.qt_viewer.dockLayerControls.toggleViewAction().trigger()
+# in napari 0.4.13 it will become private:
+# viewer.window._qt_viewer.dockLayerControls.toggleViewAction().trigger()
+# but maybe a better public method will come for this
+
+viewer.theme = "light"
+viewer.dims.ndisplay = 3
+viewer.axes.visible = True
+viewer.axes.colored = False
+viewer.axes.labels = False
+viewer.text_overlay.visible = True
+viewer.text_overlay.text = "Hello World!"
+
+myblob = data.binary_blobs(
+    length=200, volume_fraction=0.1, blob_size_fraction=.3,  n_dim=3, seed=42)
+myoctahedron = octahedron(100)
+myball = ball(100)
+
+files = {
+    "blob": myblob,
+    "ball": myball,
+    "octahedron": myoctahedron,
+}
+
+viewer.add_labels(myball, name='result') # better way to init empty labels?
+viewer.camera.angles = (19, -33, -121)
+# viewer.camera.zoom = 1.7 why not here?
```

Comment:
I think when you set the data below you trigger a refresh which might be resetting the camera zoom? I'm not sure, I havn't looked into to it, but we can investigate

Reply:
I've just tested this code again after three months, and the zoom level does actually not reset anymore.
So I can use this line here :)